torch/testing/_asserts.py

```diff
@@ -785,21 +786,82 @@ def assert_close(
     | other                     | ``0.0``    | ``0.0``  |
     +---------------------------+------------+----------+
 
-    The namespace that will be passed to :attr:`msg` if its a callable comprises the following attributes:
+    The namespace of diagnostic information that will be passed to :attr:`msg` if its a callable has the following
+    attributes:
 
-    - total_elements (int): Total number of values.
-    - total_mismatches (int): Total number of mismatches.
-    - mismatch_ratio (float): Quotient of total mismatches and total elements.
-    - max_abs_diff (Union[int, float]): Greatest absolute difference of the inputs.
-    - max_abs_diff_idx (Union[int, Tuple[int, ...]]): Index of greatest absolute difference.
-    - max_rel_diff (Union[int, float]): Greatest relative difference of the inputs.
-    - max_rel_diff_idx (Union[int, Tuple[int, ...]]): Index of greatest relative difference.
+    - ``number_of_elements`` (int): Number of elements in each tensor being compared.
+    - ``total_mismatches`` (int): Total number of mismatches.
+    - ``mismatch_ratio`` (float): Total mismatches divided by number of elements.
+    - ``max_abs_diff`` (Union[int, float]): Greatest absolute difference of the inputs.
+    - ``max_abs_diff_idx`` (Union[int, Tuple[int, ...]]): Index of greatest absolute difference.
+    - ``max_rel_diff`` (Union[int, float]): Greatest relative difference of the inputs.
+    - ``max_rel_diff_idx`` (Union[int, Tuple[int, ...]]): Index of greatest relative difference.
 
     For ``max_abs_diff`` and ``max_rel_diff`` the type depends on the :attr:`~torch.Tensor.dtype` of the inputs.
 
-    .. seealso::
-
-        To assert that the values of a tensor pair are bitwise equal, use :func:`assert_equal` instead.
+    Examples:
+        >>> expected = torch.tensor([1e0, 1e-1, 1e-2])
+        >>> actual = torch.acos(torch.cos(expected))
+        >>> torch.testing.assert_close(actual, expected)
+
+        >>> expected = torch.tensor([1e0, 1e-1, 1e-2])
+        >>> actual = torch.acos(torch.cos(expected))
+        >>> torch.testing.assert_close(actual, expected, rtol=1e-5, atol=0.0)
+        AssertionError: Tensors are not close!
+
+        >>> import math
+        >>> expected = math.sqrt(2.0)
+        >>> actual = 2.0 / math.sqrt(2.0)
+        >>> torch.testing.assert_close(actual, expected)
+
+        >>> import numpy as np
+        >>> expected = np.array([1e0, 1e-1, 1e-2])
+        >>> actual = np.arccos(np.cos(expected))
+        >>> torch.testing.assert_close(actual, expected)
+
+        >>> # The type of the sequences does not have to match. They only have to have the same
```

Comment:
This comment is great but add another one before it explaining that this is demonstrating sequence x sequence comparison